examples/live/run.sh

```diff
@@ -34,6 +34,29 @@ pycbc_hdf5_splitbank \
 mv template_bank_0.hdf template_bank.hdf
 rm -f template_bank_*.hdf
 
+echo -e "\\n\\n>> [`date`] Generating injection"
+
+pycbc_generate_hwinj \
+  --network-snr 20.0 \
+  --ra 45.0 \
+  --dec 45.0 \
+  --polarization 0.0 \
+  --approximant SEOBNRv4 \
+  --mass1 25.0 \
+  --mass2 25.0 \
+  --inclination 0.0 \
+  --taper TAPER_START \
+  --waveform-low-frequency-cutoff 10 \
+  --geocentric-end-time 1272790440 \
+  --instruments H1 L1 V1 \
+  --low-frequency-cutoff 10 \
+  --sample-rate H1:16384 L1:16384 \
+  --gps-end-time 1272790500 \
+  --gps-start-time 1272790000 \
+  --psd-model H1:aLIGOMidLowSensitivityP1200087 L1:aLIGOMidLowSensitivityP1200087 V1:AdVEarlyLowSensitivityP1200087
+  
+gunzip hwinjcbc_1272790426.xml.gz
```

Comment:
I think PyCBC's injection code can read gzipped XML, so this should not be needed.